### DISCUSSION TITLE
"/opt/python/lib/python3.10/site-packages/cryptography/hazmat/bindings/_rust.abi3.so: cannot open shared object file: No such file or directory",

### DISCUSSION THREAD

**Discussion by Ijazmuhammed** [keithrozario/Klayers#459] [Q&A]
I am working on processing the data from source to snowflake using lambda. I was facing an issue with pyarrow and snowflake connector module while uploading as layer since both have an unzip size of more than 250 mb . Now i have taken the arn for pyarrow from klayers and now facing this "errorMessage": "/opt/python/lib/python3.10/site-packages/cryptography/hazmat/bindings/_rust.abi3.so: cannot open shared object file: No such file or directory", while writing to snowflake. since klayers doesnt have an arn for snowflake connector python i created the layer for x86-64 for python 3.9 . The particular file is present in the folder. But not able to detect it. I have raised pr for getting arn for snowflake lambda layer. what could be the issue of it. Tried so many things but not working.

**Comment by keithrozario** [comment]
Sorry I never got around to answering this. There is a hard limit around layer versions unfortunately, which can be overcome via packaging the entire thing as a container.
